framework/format/format.h

```diff
@@ -312,6 +313,8 @@ struct ResizeWindowCommand2
     uint32_t         pre_transform;
 };
 
+// This command header's "usage" member is sized incorrectly.  Existing decoding must remain in place in order to
+// process captures that already exist.  Do not use this command header in any new code.
```

Comment:
Just an idea with no strong opinion on adapting it:
Perhaps the _new_ should be named `CreateHardwareBufferCommandHeader` and the _existing_ one `CreateHardwareBufferCommandHeader_deprecated` or something along those lines?

Reply:
Yes, I'll do that.